Lib/urllib/parse.py

```diff
@@ -38,29 +39,36 @@
            "DefragResult", "ParseResult", "SplitResult",
            "DefragResultBytes", "ParseResultBytes", "SplitResultBytes"]
 
-# A classification of schemes ('' means apply by default)
-uses_relative = ['ftp', 'http', 'gopher', 'nntp', 'imap',
+# A classification of schemes.
+# We use '' as the scheme value for default scheme in urlsplit and urlparse.
+
+uses_relative = ['', 'ftp', 'http', 'gopher', 'nntp', 'imap',
```

Comment:
Ah, I misunderstood the comment. I thought you were listing the functions that the lists affected. What about something like

The empty string classifies relative URLs with no scheme specified, being the default returned by “urlsplit” and “urlparse”